.travis.yml

```diff
@@ -5,7 +5,7 @@ jdk:
 services:
 - docker
 node_js:
-- "8.9.3"
```

Comment:
Would it make sense to upgrade Node.Js to 10 or higher? LTS for 8 is ending in January: https://nodejs.org/en/about/releases/

Reply:
@harbu 
```
% node --version
v10.16.3
% npm --version
6.9.0
```